K/kcp_jll/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "kcp_jll"
+version = v"1.5.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/skywind3000/kcp.git", "87c0e6a92bb7d8b49937522cdd5844905d58d8e6"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd kcp/
+cat CMakeLists.txt | sed 's/STATIC/SHARED/' | sed 's/ARCHIVE/LIBRARY/' > CMakeLists.txt.new
+mv CMakeLists.txt.new CMakeLists.txt
```

Comment:
For the record, you can do this with a single command:
```suggestion
sed -i -e 's/STATIC/SHARED/' -e sed 's/ARCHIVE/LIBRARY/' CMakeLists.txt
```